consume_test.go

```diff
@@ -978,8 +978,8 @@ func (reg *testRegistry) fakeServerHandler(w http.ResponseWriter, r *http.Reques
 			panic(err)
 		}
 	}
-	w.WriteHeader(http.StatusOK)
 	w.Header().Set("Content-Type", "application/vnd.schemaregistry.v1+json")
+	w.WriteHeader(http.StatusOK)
```

Comment:
🌷 what's the reason for that?

Reply:
From https://golang.org/pkg/net/http/:

```
// Header returns the header map that will be sent by
// WriteHeader.
```
So `Content-Type` wasn't written so the test failed with:
```
✦4 ❯ go test
--- FAIL: TestConsumeAvroMessage (0.01s)
    consume_test.go:885:
        error:
          got non-nil error
        got:
          e`invalid value: cannot get schema for id 1: cannot unmarshal JSON response from http://127.0.0.1:33367/schemas/ids/1: unexpected content type text/plain; want application/json; content: {"schema":"{\"fields\":[{\"default\":0,\"name\":\"A\",\"type\":\"long\"},{\"default\":0,\"name\":\"B\",\"type\":\"long\"}],\"name\":\"record\",\"type\":\"record\"}"}`
        stack:
          /home/quique/dev/go/src/github.com/heetch/hkt/consume_test.go:885
            c.Assert(err, qt.IsNil)
```
Because `Content-Type` is the default one `text/plain`. `WriteHeader()` should be called after `Header()` settings.